must-gather/collection-scripts/gather_nodes

```diff
@@ -120,7 +120,7 @@ for NODE in $(oc get nodes --no-headers -o custom-columns=':metadata.name'); do
         oc adm node-logs "$NODE" -u "$UNIT" > "${NODE_PATH}/${NODE}_logs_$UNIT" &
     done
 
-    oc debug node/${NODE} -- sh -c "(chroot /host rpm -qa | egrep '(kata-containers|qemu)' | sort)" > "${NODE_PATH}/version"
+    oc debug node/${NODE} -- sh -c "(chroot /host rpm -qa | egrep '(kata-containers|qemu|virtiofsd)' | sort)" > "${NODE_PATH}/version"
```

Comment:
Maybe `s/qemu/qemu-kvm-core/` for a more precise output, otherwise we might catch other RPMs that we don't really care for (e.g. `ipxe-roms-qemu` or `qemu-guest-agent`).

Reply:
I'm going to leave all the rpms